src/controllers/deliveryLog.js

```diff
@@ -74,3 +74,14 @@ export const updateLogById = async (req, res) => {
   })
   sendDataResponse(res, 201, { log: updatedLog })
 }
+
+export const getAllLogs = async (req, res) => {
+  const exerciseId = +req.query.exerciseId
+  let logs = await dbClient.deliveryLog.findMany()
+  if (!isNaN(exerciseId)) {
+    logs = await dbClient.deliveryLog.findMany({
+      where: { exerciseId }
+    })
+  }
```

Comment:
This is always going to run 2 queries if the exercise ID is provided, which is an unnecessary database operation. You should only perform either the `findMany()` *or* the `findMany({ where: ... })`